src/flightlog_fielddefs.js

```
@@ -38,660 +39,1130 @@ export const FlightLogEvent = makeReadOnly({
+export const FF_AVERAGING = makeReadOnly(['OFF', '2_POINT', '3_POINT', '4_POINT'])
+
+export const SIMPLIFIED_PIDS_MODE = makeReadOnly(['OFF', 'ON - RP', 'ON - RPY'])
+
+export const THROTTLE_LIMIT_TYPE = makeReadOnly(['OFF', 'SCALE', 'CLIP'])
 
 export function adjustFieldDefsList(firmwareType, firmwareVersion) {
-  if (
-    firmwareType === FIRMWARE_TYPE_BETAFLIGHT &&
-    semver.gte(firmwareVersion, "3.3.0")
-  ) {
+  if (firmwareType == FIRMWARE_TYPE_ROTORFLIGHT) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)